docs/GHA.md

```diff
@@ -12,17 +12,3 @@ In a GitHub Actions workflow, you can set these environmental variables using th
       core.exportVariable('ACTIONS_CACHE_URL', process.env.ACTIONS_CACHE_URL || '');
       core.exportVariable('ACTIONS_RUNTIME_TOKEN', process.env.ACTIONS_RUNTIME_TOKEN || '');
 ```
-
-To write to the cache, set `SCCACHE_GHA_CACHE_TO` to a cache key, for example
-`sccache-latest`. To read from cache key prefixes, set `SCCACHE_GHA_CACHE_FROM`
-to a comma-separated list of cache key prefixes, for example `sccache-`.
-
-In contrast to the [`@actions/cache`](https://github.com/actions/cache) action, which saves a single large archive per cache key, `sccache` with GHA cache storage saves each cache entry separately.
-
-GHA cache storage will create many small caches with the same cache key, e.g. `SCCACHE_GHA_CACHE_TO` and `SCCACHE_GHA_CACHE_FROM`. These GHA caches are differentiated by their [_version_](https://github.com/actions/cache#cache-version). The GHA cache implementation in `sccache` calculates the cache version from the [`sccache` entry key](docs/Caching.md), e.g. the source file path.
-
-For example, if a cache entry has the version `main.rs` and has GHA cache entries for the `sccache-1` and `sccache-2` keys, then `SCCACHE_GHA_CACHE_FROM=sccache-` will match both and [return the most recent entry](https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows#matching-a-cache-key).
-
-This behavior is useful for scoping caches from different versions of Rust or for cross-platform builds (`rust-sdk-{RUST_TOOLKIT}-{TARGET_TRIPLE}-`), and to allow newer commits to override older caches by adding the Git SHA as a suffix (`-{GITHUB_SHA}`), as in the following screenshot.
```

Comment:
Without the option of reading caches from a cache key prefix, PR builds cannot use caches from the main branch and will never use caches from previous builds.

Reply:
User can specify the prefix by setting `root`.

Reply:
That's not now GitHub Actions cache works...

Reply:
> That's not now GitHub Actions cache works...

Can explain a bit to me? I'm not very familiar with github action cache services.

So far, I just treat github action cache as a storage service which means:

- We don't care about how github action cache works (especially for `actions/cache`)
- We don't want to work along with caches that written by others.

Sccache will calculate the unique path for every object file, and we just use the same path as the cache key. Will this behavior break our cache service?

Reply:
> That's not now GitHub Actions cache works...

Can explain a bit to me? I'm not very familiar with github action cache services.

So far, I just treat github action cache as a storage service which means:

- We don't care about how github action cache works (especially for `actions/cache`)
- We don't want to work along with caches that written by others.

Sccache will calculate the unique path for every object file, and we just use the same path as the cache key. Will this behavior break our cache service?

Reply:
@jakelee8 sorry for the dumb question but what is wrong here? thanks

Reply:
GitHub Actions cache isn't a simple key-value store. See https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows

Reply:
Maybe it worth to start a discussion for this~

Reply:
Take this action as an example https://github.com/mozilla/sccache/actions/runs/3868518097/jobs/6594049385

The first run on this PR will have cache hits which means we are reading cache from main branch.